apis/python/test/test_object_index.py

```diff
@@ -251,10 +251,16 @@ def test_object_index(tmp_path):
             index_type=index_type,
             object_reader=reader,
             embedding=embedding,
+            num_subspaces=4,
         )
 
         # Check initial ingestion
         index.update_index(partitions=10)
+
+        # TODO(SC-48908): Fix IVF_PQ with object index queries and remove.
```

Comment:
Is this failing?

Reply:
Yep, this fails - we do get back valid IDs in `objects["external_id"]`, but they are in a different order than expected. So was going to fix this in a follow-up if that's okay with you?
```
distances, objects, metadata = index.query(
        {"object": np.array([[dim_id, dim_id, dim_id, dim_id]])},
        metadata_array_cond=f"test_attr >= {dim_id}",
        k=5,
        **query_kwargs,
    )
    assert np.array_equiv(
        objects["external_id"], np.array([v_id, v_id + 1, v_id + 2, v_id + 3, v_id + 4])
    )
```